File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/subscope/RepeatUsageDependency_ViewModelTestSetupImpl.cpp
#include "RepeatUsageDependency_ViewModelTestSetupImpl.h"
#include <memory>
#include "RepeatUsageDependency_ViewModel.h"
#include "MessageBus.h"
#include "dependencies/Logger.h"

namespace dependencies::subscope
{
  void RepeatUsageDependency_ViewModelTestSetupImpl::Init()
  {
  }
  std::shared_ptr<RepeatUsageDependency_ViewModel> RepeatUsageDependency_ViewModelTestSetupImpl::BuildSut()
  {
    return nullptr;
  }
  std::shared_ptr<MessageBus> RepeatUsageDependency_ViewModelTestSetupImpl::getMessageBus()
  {
    return nullptr;
  }
  std::shared_ptr<Logger> RepeatUsageDependency_ViewModelTestSetupImpl::getLogger()
  {
    return nullptr;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/subscope/RepeatUsageDependency_ViewModelTestSetupImpl.cs
/// <filename>
///     RepeatUsageDependency_ViewModelTestSetupImpl.cs
/// </filename>
using dependencies.subscope;
using dependencies;

namespace dependencies.subscope
{
    internal class RepeatUsageDependency_ViewModelTestSetupImpl : dependencies.subscope.RepeatUsageDependency_ViewModelTestSetup
    {
        public override void Init()
        {
        }

        public override dependencies.subscope.RepeatUsageDependency_ViewModel BuildSut()
        {
            return null;
        }

        public override dependencies.subscope.MessageBus getMessageBus()
        {
            return null;
        }

        public override dependencies.LoggerSpy getLogger()
        {
            return null;
        }
    }
}
